src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/EXUIIACMC.scala

```diff
@@ -159,10 +236,11 @@ object EXUIIACMC {
 * Below group contains all the requests for Appealant address search
 ======================================================================================*/
     
-    .group("XUI${service}_110_StartAppealDetailsAddressSearch") {
-      exec(http("XUI${service}_110_StartAppealDetailsAddressSearch")
+    .group("XUI${service}_120_StartAppealDetailsAddressSearch") {
+      exec(http("XUI${service}_120_StartAppealDetailsAddressSearch")
         .get("/api/addresses?postcode=TW33SD")
```

Comment:
It would be good to vary to data such as postcodes. The Probate code uses a common function for the postcode lookup (based on a csv) and generates address lines which can be used in the corresponding JSON files.